internal/webhook/external/refresh.go

```diff
@@ -26,6 +30,7 @@ func refreshWarehouses(
 	w http.ResponseWriter,
 	c client.Client,
 	project string,
+	qualifiers []string,
 	repoURLs ...string,
```

Comment:
I know that the way we landed on the current signature for this function was incremental. We used to accept just one qualifier, but multiple repoURLs and that's why repoURLs originally came last -- because you can only have one variadic argument and it must be last...

But now that we accept multiple qualifiers in addition to the multiple urls, I feel like it would be more intuitive for repoURLs to come before qualifiers.

Could we swap the order here?

```suggestion
	repoURLs []string,
	qualifiers ...string,
```

This is non-blocking, but I do think it's a good idea.